server.py

```diff
@@ -0,0 +1,149 @@
+import json
```

Comment:
Please run pep8 compatible linters such as flake8 and pylint on code file as there a number of format issues in this one. Alternatively use python black to autoformat in an opinionated way.